### ISSUE TITLE
Do not crash when rendering alt text for missing image if value of align is invalid

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor-pdf#1781]
If the value of the align attribute on an image block is invalid (e.g., middle), and the image is missing, the converter will crash. It should ignore the invalid value in this case.
